learning_journal/models.py

```diff
@@ -18,10 +22,25 @@
 Base = declarative_base()
 
 
-class MyModel(Base):
-    __tablename__ = 'models'
+class Entry(Base):
+    __tablename__ = 'entries'
     id = Column(Integer, primary_key=True)
-    name = Column(Text)
-    value = Column(Integer)
+    title = Column(Unicode(length=255), nullable=False, unique=True)
+    body = Column(Unicode)
+    created = Column(DateTime, default=datetime.utcnow)
+    edited = Column(DateTime, onupdate=datetime.utcnow)
```

Comment:
This column will not have a value until the object is first updated (not when first created).  Can you make it so it *both* has an initial value *and* gets updated each time the object is altered?